bin/nvcc_wrapper

```diff
@@ -308,6 +308,16 @@ do
   shift
 done
 
+#Check if nvcc exists
+if [ $host_only -ne 1 ]; then
+  var=$(which nvcc )
+  if [ $? -gt 0 ]
+  then
+    exit $?
```

Comment:
Thanks for working on this!  Could you please add an error message here, like "Could not find nvcc in PATH"?

Reply:
Thanks @mhoemmen . Error message added.